Testing/Unit/Python/sitkBasicFilterTests.py

```diff
@@ -135,6 +135,41 @@ def test_landmark_based_transform_initializer(self):
         self.assertEqual(tx_initializer.Execute(sitk.AffineTransform(3)).__class__,
                         sitk.AffineTransform)
 
+
+    def test_centered_transfrom_initializer(self):
+        """Test CenteredTransformInitializer return Transform is downcasted"""
```

Comment:
'Test CenteredTransformInitializer to check that the return Transform is properly downcasted'

It took me a sec to parse what your description was saying.

Reply:
done